### PR TITLE
[helm] add pruner config

### PR DESCRIPTION
**Pull request by sherry-x**
add config to tune pruner window

### CODE REVIEW

File: terraform/helm/validator/files/configs/fullnode.yaml [hunk 1]
```diff
@@ -21,6 +21,8 @@ execution:
 
 storage:
   backup_service_address: "0.0.0.0:6186"
+  storage_pruner_config:
+    default_prune_window: {{ .Values.validator.config.default_prune_window | default 10000000 }}
```
Comment (sitalkedia) [inline terraform/helm/validator/files/configs/fullnode.yaml:25]: Let's make this a billion just to be on the safe side?

Comment (sitalkedia) [reply terraform/helm/validator/files/configs/fullnode.yaml:25]: Just to be clear, 10 M is the current default value in code, we want to bump it to 1 B over here.

Comment (sherry-x) [reply terraform/helm/validator/files/configs/fullnode.yaml:25]: @sitalkedia the override will happen on the terraform side, each testnet can customize their own value from there, here I just matched the code default

Comment (sitalkedia) [reply terraform/helm/validator/files/configs/fullnode.yaml:25]: Ah, I see. Thanks for clarifying.

**Review by sitalkedia — APPROVED** [review]

File: terraform/helm/validator/files/configs/fullnode.yaml (hunk 1 above)
Comment (sherry-x) [reply terraform/helm/validator/files/configs/fullnode.yaml:25]: @sitalkedia the override PR  https://github.com/aptos-labs/internal-ops/pull/147

**Review by rustielin — APPROVED** [review]

**Comment by sherry-x** [comment]
@sitalkedia seems if we provide the config here we need to provide all three, could you please take another look at other default value I set here? thanks!

File: terraform/helm/validator/values.yaml [hunk 2]
```diff
@@ -31,6 +31,9 @@ validator:
     sync_only: false
     mempool_poll_count:
     round_initial_timeout_ms:
+    default_prune_window: 10000000
+    state_store_prune_window: 0
```
Comment (sitalkedia) [inline terraform/helm/validator/values.yaml:35]: Let's set the default to what we have in code `1000000`,

File: terraform/helm/validator/values.yaml [hunk 3]
```diff
@@ -31,6 +31,9 @@ validator:
...
+    max_version_to_prune_per_batch: 100
```
Comment (sitalkedia) [inline terraform/helm/validator/values.yaml:36]: This looks good to me.

**Review by sitalkedia — CHANGES_REQUESTED** [review]

File: terraform/helm/validator/values.yaml (hunk 2 above)
Comment (sherry-x) [reply terraform/helm/validator/values.yaml:35]: @sitalkedia default is 10M, 7 zeros

Comment (sitalkedia) [reply terraform/helm/validator/values.yaml]: @sherry-x - That's the default_pruner_window. For state_store_pruner_window, the default value is 1M - https://github.com/aptos-labs/aptos-core/blob/main/config/src/config/storage_config.rs#L101

**Review by sitalkedia — APPROVED** [review]

**Pull request closed without merging**